WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/GutenbergLayoutPickerViewController.swift

```diff
@@ -118,6 +120,7 @@ class GutenbergLayoutPickerViewController: UIViewController {
     }
 
     var completion: PageCoordinator.TemplateSelectionCompletion? = nil
+    private var isUpdating: Bool = false
```

Comment:
This isn't used AFAIK.

Reply:
Good catch this was added then the need was removed by a later commit so it slipped through the cracks. Good eye on catching that cleanup.